src/style.css

```diff
@@ -1 +1,81 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir.ttc);
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+.search {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 20px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  transition: box-shadow 0.3s ease;
+}
+
+.search:hover,
+.search:focus-within {
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
```

Comment:
While using rgba for the box-shadow color is fine, it's a good practice to be consistent with color formats throughout your CSS. You've used hex color codes for borders and shadows elsewhere, so consider using the rgba equivalent of #3d4e611a for consistency.